src/GraphQL/Types/Schema.cs

```diff
@@ -128,37 +84,42 @@ public IEnumerable<DirectiveGraphType> Directives
             get => _directives;
             set
             {
-                if (value == null)
```

Comment:
Note that I changed the behavior. It seemed strange to me that passing `null` does not clear the collection.